lib/src/common/disposable.dart

```diff
@@ -284,6 +300,14 @@ class Disposable implements _Disposable, DisposableManagerV3 {
       _logger.info(
           '$runtimeType $hashCode took ${stopwatch.elapsedMicroseconds / 1000000.0} seconds to dispose');
     }
+
+    flagLeak(runtimeType.toString());
+  }
+
+  @mustCallSuper
+  @override
+  void flagLeak(String description) {
+    _leakFlag = new LeakFlag(description);
```

Comment:
just to verify, this wouldn't _need_ to be overridden by consumers of Disposable, right?